mcs/class/corlib/Test/System/DecimalTest.cs

```diff
@@ -882,6 +882,8 @@ public void TestRound ()
 				{1.234567890m, 2, 1.23m}, {1.23450000001m, 3, 1.235m}, 
 				{1.2355m, 3, 1.236m}, 
 				{1.234567890m, 4, 1.2346m}, {1.23567890m, 2, 1.24m}, 
+				{1.00567890m, 2, 1.01m},
+				{1.005m, 2, 1.01m},
```

Comment:
It's a bug though, right?

This is how ruby does it (round takes # digits (not decimals)):

``` ruby
0.005.round(2)
# => 0.01
```

Clojure:

``` clojure
(format "%.2f" 1.005)
; => "1.01"
```